arm-servermanagement/2015-07-01-preview/servermanagement.json

```diff
@@ -1633,12 +1607,40 @@
           "type": "number",
           "description": "The available memory on the gateway host machine in megabytes."
         },
-        "gatewayCpuPercent": {
+        "gatewayCpuUtilizationPercent": {
           "type": "number",
```

Comment:
Do you expect percentage in float/decimal ? If so then add `"format": "decimal" / "float"` as applicable.
If you expect the percentage number to be integers then this should be `"type": "integer", "format": "int32"`